hudi-common/src/main/java/org/apache/hudi/metrics/Metrics.java

```diff
@@ -99,8 +97,7 @@ public static synchronized void shutdownAllMetrics() {
   private List<MetricsReporter> addAdditionalMetricsExporters(HoodieMetricsConfig metricConfig) {
     List<MetricsReporter> reporterList = new ArrayList<>();
     List<String> propPathList = StringUtils.split(metricConfig.getMetricReporterFileBasedConfigs(), ",");
-    try (HoodieStorage storage = HoodieStorageUtils.getStorage(
-        propPathList.get(0), HadoopFSUtils.getStorageConf(new Configuration()))) {
+    try (HoodieStorage storage = HoodieStorageUtils.getStorage(propPathList.get(0))) {
```

Comment:
Separate this change out?  Also, should it get the actual config from somewhere else?  An empty config may not work in some cases (reach out to the author who made the changes in `Metrics` class).

Reply:
I don't see how this is any different

Reply:
don't make it use hoodiestorageutils becuase we don't want to expose an option to not pass a config. Create followup ticket

Reply:
https://issues.apache.org/jira/browse/HUDI-7731

Reply:
The empty `new Configuration()` should not be used here.  We should track down the existing code since it may cause issues of accessing the files.